Day-I-Social-Card/social-card/src/components/FooterComponents/Footer.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './Footer.css';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+const CardFooter = (props) => {
+    let likes = 0;
```

Comment:
CardFooter should be converted to a class component to manage likes and repost in `state`